certified-connectors/Africa's Talking Airtime/apiDefinition.swagger.json

```diff
@@ -0,0 +1,303 @@
+                        "description": "Phone number to be topped up in international format (e.g +234811222333)",
+                        "title": "Phone Number",
+                        "x-ms-visibility": "important"
+                      },
+                      "amount": {
+                        "type": "string",
+                        "description": "The value of airtime to send together with the currency code (e.g. KES 10.00).",
+                        "title": "Amount",
+                        "x-ms-visibility": "important"
+                      }
+                    },
+                    "required": [
+                      "amount",
+                      "phoneNumber"
+                    ]
+                  },
+                  "description": "recipients"
+                }
+              },
+              "x-ms-visibility": "important",
+              "required": [
+                "recipients",
+                "username"
+              ]
+            },
+            "x-ms-visibility": "important"
+          }
+        ]
+      }
+    },
+    "/query/transaction/find": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "type": "string",
+                  "description": "status"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Find transaction status",
+        "description": "Find a particular airtime transaction's status",
+        "operationId": "FindTransactionStatus",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "username",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Your application's username",
+            "x-ms-summary": "Username",
+            "x-ms-visibility": "important"
+          },
+          {
+            "name": "transactionId",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "ID of the transaction you'd like to find.",
+            "x-ms-summary": "Transaction ID",
+            "x-ms-visibility": "important"
+          }
+        ]
+      }
+    },
+    "/api/v1/powerapps/{username}/airtime/status/callback": {
+      "x-ms-notification-content": {
+        "schema": {
+          "type": "object",
+          "properties": {
+            "requestId": {
+              "type": "string",
+              "description": "requestId"
+            },
+            "status": {
+              "type": "string",
+              "description": "status"
+            }
+          }
+        },
+        "description": "A new airtime delivery status notification"
+      },
+      "post": {
+        "responses": {
+          "200": {
+            "description": "200"
```

Comment:
@crispinkoech Can you please provide a valid description for the "200" response code instead of "200"